### DISCUSSION TITLE
General-Purpose Web Integration

### DISCUSSION THREAD

**Discussion by tom-ricci** [coollabsio/coolify#2626] [Service Template Requests]
It would be great to have a general-purpose framework for integrating other websites in the Coolify dashboard, allowing a team to use Coolify as a general dashboard for everything related to their servers. 

**Possible use cases:**
1. You own your hardware, and want to manage the settings of the LAN its connected to. Instead of running a proxy on one of the servers and accessing OPNsense or pFsense or whatever through the proxy, you could open it inside a page directly in Coolify.
2. You're running Supabase on a server managed by Coolify, and you want to manage your Supabase instance in the same dashboard as the server itself.
3. You're running a lot of applications with a lot of different TLDs, making wildcard DNS unfeasible. You could open your DNS settings in Coolify and manage your domains right there.
4. You have some regulatory requirement (idk, HIPPA or something) or insurance requirement that prevents certain information from being accessible publicly, and need to use a proxy to access it.

You could probably just do this with an iframe that resolves to `your.coolify.instance/some-specific-integrations-path?url=https://your.proxied/url/here` which Traefik notices and proxies to whatever the query string says.

This may also be a path to supporting plugins in the future.

**Comment by CarlQLange** [comment]
I'm investigating Coolify for some future work and I'm curious if anything has moved on this in the last year or so?
